farnaz/challenge1.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>challenge1</title>
+</head>
+
+<body>
+    <script>
+        //loop through all numbers between 0 and 100
+        for (let i = 0; i <= 100; i++) {
+            let isPrime = true;
+
+            //Check if the number is divisible by any number other than 1 and itself
+            for (let j = 2; j < i; j++) {
+                if (i % j === 0) {
+                    isPrime = false;
```

Comment:
What is the purpose of the `isPrime` variable?

Reply:
isPrime is a variable used to store a boolean value (true or false). If it is a prime number, it is true and if it is not a prime number, it is false.